src/sentry/models/projectkey.py

```diff
@@ -36,8 +36,8 @@ class ProjectKeyStatus(object):
 class ProjectKey(Model):
     project = FlexibleForeignKey('sentry.Project', related_name='key_set')
     label = models.CharField(max_length=64, blank=True, null=True)
-    public_key = models.CharField(max_length=32, unique=True, null=True)
-    secret_key = models.CharField(max_length=32, unique=True, null=True)
+    public_key = EncryptedCharField(max_length=32, unique=True, null=True)
+    secret_key = EncryptedCharField(max_length=32, unique=True, null=True)
```

Comment:
For a block cipher like AES the size will be whatever the size of the input is rounded up to 8 bytes (or if it's already a multiple of 8 bytes, with an extra 8 bytes on the end). For fernet there's also additional overhead for things like the version tag, the authentication tag, the IV, etc. 

For a stream cipher, the output will be equal to the input in size (plus any of the additional overhead like above).

Given all of that, I agree with @mattrobenolt you should just use a ``text`` type and a ``unique`` index on the hashed fields.